docs/root/configuration/observability/access_log.rst

```diff
@@ -353,6 +353,16 @@ The following command operators are supported:
   TCP
     Not implemented ("-").
 
+%FILTER_STATE(KEY):Z%
+  HTTP
+    :ref:`Filter State <arch_overview_data_sharing_between_filters>` info, where the KEY is required to
+    look up the filter state object. The serialized proto will be logged as JSON string if possible.
+    If the serialized proto is unknown to Envoy it will be logged as protobuf debug string.
```

Comment:
How can this happen? All these objects are produced by Envoy, it must have the descriptors, right?

Reply:
a wasm serialized Any that envoy doesn't have descriptor.